src/node/http.ts

```diff
@@ -240,16 +240,17 @@ export abstract class HttpProvider {
     response: HttpStringFileResponse,
     extraOptions?: Omit<T, "base" | "csStaticBase" | "logLevel">,
   ): HttpStringFileResponse {
+    const base = this.base(route)
     const options: Options = {
-      base: this.base(route),
-      commit: this.options.commit,
+      base,
+      csStaticBase: base + "/static/" + this.options.commit + this.rootPath,
```

Comment:
Oh nvm, ah I see, the commit is still here but we store the base instead of just the commit.